### DISCUSSION TITLE
Asking for custom Domain on GitHub Pages

### DISCUSSION THREAD

**Discussion by Sujal-2820** [community/community#31839] [Other Feature Feedback, Questions, & Ideas]
Hi There,
I am facing ana issue with the GitHub pages. As soon as I click the GitHub Pages button, I get a dropbox saying "verify your domain". While I do not want any custom domain. I just want to Host my website through GitHub. Please help me get out of this issue!!

**Comment by radar-blip** [comment]
You may be looking at the wrong Pages settings. 
There is another setting for Github Pages once you open up a repository.

**Comment by jfullstackdev** [comment]
seems this is fixed by now 

![image](https://user-images.githubusercontent.com/47092464/188315815-cf79cf48-a853-4a01-af20-5ff831f08b85.png)

**Comment by joelibaceta** [comment]
To set a custom domain you should add some DNS records to your domain in order to delegate the domain resolution to github.

> Check this https://docs.github.com/es/pages/configuring-a-custom-domain-for-your-github-pages-site
